api/Element.json

```diff
@@ -3153,28 +3153,59 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/requestFullscreen",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true,
+              "alternative_name": "webkitRequestFullscreen"
```

Comment:
Should APIs use the `prefix` property or `alternative_name`? In this case, Firefox implements it as `mozRequestFullScreen` (capital S) so I made all of them `alternative_name` for consistency in this specific API.

Reply:
In this case, it should be `"prefix": "webkit"`, since capitalisation isn’t different.